libctru/include/3ds/svc.h

```diff
@@ -994,7 +994,7 @@ void svcBreak(UserBreakType breakReason);
  * @param croInfo Library information.
  * @param croInfoSize Size of the above structure.
  */
-void svcBreakRO(UserBreakType breakReason, const void* croInfo, u32 croInfoSize);
+void svcBreakRO(UserBreakType breakReason, const void* croInfo, u32 croInfoSize) __asm__("svcBreak");
```

Comment:
Could you instead do:

    void svcBreakRO(UserBreakType breakReason, const void* croInfo, u32 croInfoSize);
    static inline void svcBreak(UserBreakType breakReason)
    {
      return svcBreakRO(breakReason, NULL, 0);
    }

and implement svcBreakRO in libctru/source/svc.s